docs/index.md

```diff
@@ -239,7 +239,7 @@ The following arguments are used to configure the provider:
   For on-prem deployments, this setting should not be specified.
 * `global_manager` - (Optional) True if this is a global manager endpoint.
   False by default.
-* `license_keys` - (Optional) List of NSX-T license keys. License keys are applied
+* `license_keys` - (Optional) List of NSXlicense keys. License keys are applied
```

Comment:
I think a space is missing here and below

Reply:
Resolved in [f519e3d](https://github.com/vmware/terraform-provider-nsxt/commit/f519e3dbda00146e894d30b05d1e3e2478d08eb5).